release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -178,6 +178,28 @@ With this release, you can access the NMstate Operator and resources such as the
 [id="ocp-4-14-olm"]
 === Operator lifecycle
 
+[id="ocp-4-14-olmv1"]
+==== Operator Lifecycle Manager v1 (Technology Preview)
+
+{product-title} 4.14 introduces components for a next-generation iteration of Operator Lifecycle Manager (OLM) as a Technology Preview feature. Known during this phase as OLM v1, the updated framework evolves many of the concepts that have been part of the version of OLM included in OpenShift Container Platform up to this point, referred to retroactively as OLM v0.
+
+OLM v1 improves Operator and extension lifecycle management in the following areas:
+
+* Dependencies and constraints
+* Tenant isolation
+* Simplified packaging models
+* Multi-cluster support
+
+In OLM v1, a _plain bundle_ is a static collection of arbitrary Kubernetes manifests in the YAML format. In the Technology Preview phase, cluster administrators and Operator catalog maintainers can build and publish a file-based catalog that includes a plain bundle image by completing the following procedures:
+
+. Build a plain bundle image.
+. Create a file-based catalog.
+. Add the plain bundle image to your file-based catalog.
+. Build your catalog as an image.
+. Publish your catalog image.
+
+For more information, see "Managing catalogs for OLM v1".
```

Comment:
`xref` to content to be added after https://github.com/openshift/openshift-docs/pull/63086 merges.